bin/puppet_install.sh

```diff
@@ -38,11 +38,16 @@ setup_redhat() {
 }
 
 setup_fedora() {
+  release=$1
+  if [[ $release == '27' ]]
```

Comment:
ugh

Reply:
At the moment there are repos 'till fedora 26, but it works on 27. https://yum.puppetlabs.com/